index.html

```diff
@@ -0,0 +1,98 @@
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  </head>
+<style>
+    @media all {
+        #features ul,
+        #nav-bar {
+            display: flex;
+            flex-direction: row;
+            align-content: space-between;
+            align-items: center;
+        }
+        h1 {
+          margin-top: 40px;
+        }
+        header {
+            max-width: 60%;
+            margin: auto;
+        }
+        #nav-bar {
+          position: fixed;
+          top: 0;
+          background: black;
+          color: white;
+          width: 100%;
+        }
+
+        a {
+          color: white;
+        }
+        ul {
+          list-style-type: none;
+        }
+        li {
+          margin-right: 30px;
+        }
+        #nav-bar a, ul li {
+          align-self: center;
+        }
+        
+    }
+</style>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <h1>NIKE Hyper</h1>
+      <nav id="nav-bar">
+        <a class="nav-link" href="#intro">First clickable</a>
+        <a class="nav-link" href="#features">Second clickable</a>
+        <a class="nav-link" href="#demand">Third clickable</a>
+      </nav>
+      <img id="header-img" src="" alt="">
+      <section id="intro">
+          <h2>PRODUCT INTRO</h2>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+            in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui
+            officia deserunt mollit anim id est laborum.</p>
+      </section>
+      <section id="features">
+            <ul>
+              <li>Featue #</li>
+              <li>Featue #</li>
+              <li>Featue #</li>
+            </ul>
+      </section>
+      <section id="demand">
+        <form id="form" action="https://www.freecodecamp.com/email-submit" method="post">
+          <label for="email">Enter your email:</label>
+          <input type="email" id="email" name="email" placeholder="ex: john.doe@example.com" required>
+          <input type="submit" id="submit">
+        </form>
+      </section>
+      <video controls id="video" src="nike-hyperfuse-commercial.mp4"></video>
+    </header>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please remove these unnecessary comments as it may make your code unorganized and hard to maintain.